src/app/home/pages/PlansPage/types.ts

```diff
@@ -20,3 +20,16 @@ export interface IAddPlanDisabledObjModel {
   isAddPlanDisabled: boolean;
   disabledText: string;
 }
+
+export enum MigrationStepsType {
+  Prepare = 'Prepare',
+  Backup = 'Backup',
+  StageBackup = 'StageBackup',
+  StageRestore = 'StageRestore',
+  Restore = 'Restore',
+  Final = 'Final',
```

Comment:
This is now renamed to 'Cleanup' step instead of 'Final'. I also have this fixed in my PR.